content/docs/releases/release-notes/release-notes-1.16.md

```diff
@@ -7,12 +7,31 @@ cert-manager 1.16 includes various improvements to the metrics in the cert-manag
 
 ## Breaking changes
 
-1. Venafi Issuer may fail to renew Certificates if the duration conflicts with the CA minimum / maximum duration policy in Venafi.
-2. Venafi Issuer may fail to renew Certificates if the issuer has been configured for TPP with username-password authentication.
-3. Helm schema validation may reject your existing Helm values files if they contain typos or unrecognized fields.
+1. Helm schema validation may reject your existing Helm values files if they contain typos or unrecognized fields.
+   For more details, refer to the [Helm](#helm) section below.
+1. Venafi Issuer may fail to renew certificates if the requested duration conflicts with the CA’s minimum or maximum policy settings in Venafi.
+   For more details, refer to the [Venafi Issuer](#venafi-issuer) section below.
+1. Venafi Issuer may fail to renew Certificates if the issuer has been configured for TPP with username-password authentication.
+   For more details, refer to the [Venafi Issuer](#venafi-issuer) section below.
 
 ## Themes
 
+### Helm
+
+The Helm chart now includes a JSON schema which will validate the values that you supply when installing the chart.
+This will help you to get your Helm values right first time.
+It will alert you to typos and unrecognized fields in your existing Helm values files.
+And it will make it easier for the cert-manager maintainers to maintain the Helm chart,
+avoiding typos and mistakes in the default values file.
+
+> ⚠️ Helm schema validation may reject your existing Helm values files if they contain typos or unrecognized fields.
+> You can use `helm template cert-manager --repo https://charts.jetstack.io --version 1.16.x --values values.cert-manager.yaml`
```

Comment:
the `1.16.x` bit doesn't work...

Today, I can use:
helm template cert-manager --repo https://charts.jetstack.io --version 1.16.0-beta.0 --values values.yaml 

But, as a reader, I expect a command like this to work as is in the docs, and not require guessing.